src/lib/Makefile.am

```diff
@@ -20,7 +20,6 @@ librnp_la_SOURCES	= \
 	key_store.c \
 	key_store_pgp.c \
 	key_store_kbx.c \
-	key_store_ssh.c \
```

Comment:
@catap I disabled it since it needed some time to refactor it while we talked with Ronald that most likely SSH code is obsolete.

Sorry, but it's a bit insulting. The discussion was created at Jun, 28 and 10 minutes ago I edited it since found small typo.